test/Microsoft.OpenAPI.OData.Reader.Tests/Operation/EntitySetPostOperationHandlerTests.cs

```diff
@@ -264,6 +266,7 @@ internal static IEdmModel GetEdmModel(string annotation, bool hasStream = false)
         <Annotation Term=""Org.OData.Capabilities.V1.InsertRestrictions"">
           <Record>
             <PropertyValue Property=""Description"" String=""Create a new customer."" />            
+            <PropertyValue Property=""LongDescription"" String=""Create a new customer."" />            
```

Comment:
Can you add a different value here to differentiate it from the `Description` value in tests?

Reply:
On it